lib/readline.js

```diff
@@ -920,6 +921,10 @@ function emitKeypressEvents(stream) {
       var r = stream[KEYPRESS_DECODER].write(b);
       if (r) {
         for (var i = 0; i < r.length; i++) {
+          if (r[i] === '\t' && typeof r[i + 1] === 'string' && iface) {
+            iface.isCompletionEnabled = false;
```

Comment:
Ahhh sorry I didn't see the checks for it 😥 

All good to go if there is a docs note.